src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -353,8 +295,7 @@ private void determineTerritoriesToAttack(final Map<Territory, ProTerritory> att
     while (true) {
       final List<ProTerritory> territoriesToTryToAttack = prioritizedTerritories.subList(0, numToAttack);
       ProLogger.debug("Current number of territories: " + numToAttack);
-      tryToAttackTerritories(attackMap, new ProMoveOptions(), unitAttackMap, territoriesToTryToAttack,
-          transportMapList, transportAttackMap, bombardMap, new ArrayList<Unit>());
+      tryToAttackTerritories(territoriesToTryToAttack, new ArrayList<Unit>());
```

Comment:
Another passing of "new Arraylist<Unit>()" to a function for very unclear reasons. I would try to get rid of these for clarity, hard to see why it is necessary.